packages/web-components/src/components/cta-block/cta-block.scss

```diff
@@ -56,8 +53,11 @@
     @include carbon--breakpoint('md') {
       width: 90%;
     }
+  }
 
-    padding-left: 0;
+  ::slotted(#{$dds-prefix}-button-group) {
+    // <dds-button-group-item> elements have top margins
+    margin-top: -#{$carbon--layout-02};
```

Comment:
I'm not in love with this solution. Is there a way to apply style rules to deeply nested subcomponents? I.e., the following doesn't work, but is there a way to achieve its goal?

```
dds-cta-block dds-button-group[slot="action"] > dds-button-group-item {
  margin-top: 0;
}
```